common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -610,6 +610,10 @@ public static int hashCodeAscii(CharSequence bytes) {
         return new MpscLinkedQueue<T>();
     }
 
+    public static <T> Queue<T> newSpscQueue() {
+        return new SpscLinkedQueue<T>();
```

Comment:
`newMpscQueue` doesn't do that. The platform dependant thing is just `AtomicReferenceFieldUpdater` and MPSC classes have already handled it.